src/typescript/frontend/src/components/trade/OrderEntry/OrderEntryInputWrapper.tsx

```diff
@@ -5,22 +5,23 @@ export const OrderEntryInputWrapper: React.FC<
     startAdornment?: string;
     endAdornment?: string;
     labelFor?: string;
+    className?: string;
   }>
-> = ({ startAdornment, endAdornment, labelFor, children }) => {
+> = ({ startAdornment, endAdornment, labelFor, className, children }) => {
   return (
-    <div className="flex h-10 w-full items-baseline gap-2 border border-neutral-600 px-4 py-2">
-      {/* start adornment */}
+    <div
+      className={`relative flex h-10 w-full items-center border border-neutral-600 ${className}`}
```

Comment:
it renders as `undefined` in the dom.

Reply:
I usually just do this, since I've never encountered a situation where I'd want to use `undefined` as a class name.

I feel like making this slightly easier to read is worth some ugliness in the HTML that gets outputted?